scripts/globals/mobskills/red_lotus_blade.lua

```diff
@@ -24,8 +24,8 @@ mobskillObject.onMobWeaponSkill = function(target, mob, skill)
         target:showText(mob, zones[xi.zone.THRONE_ROOM].text.FEEL_MY_PAIN)
     end
 
-    local dmgmod = 1.25
-    local info = xi.mobskills.mobMagicalMove(mob, target, skill, mob:getWeaponDmg() * 4, xi.magic.ele.FIRE, dmgmod, xi.mobskills.magicalTpBonus.DMG_BONUS, 1)
+    local dmgmod = 1
+    local info = xi.mobskills.mobMagicalMove(mob, target, skill, mob:getWeaponDmg(), xi.magic.ele.FIRE, dmgmod, xi.mobskills.magicalTpBonus.NO_EFFECT, 1, 0, 1.5, 1.75, 2)
```

Comment:
Should convert this to getMobWeaponDmg if we're gonna be using it for the other weaponskill mobskills.